indico/http/client.py

```diff
@@ -155,6 +155,10 @@ def _make_request(
         if len(url_parts) > 1 and (url_parts[-1] == "json" or url_parts[-2] == "json"):
             json = True
 
+        decompress = False
+        if len(url_parts) > 1 and (url_parts[-1] == "gz"):
+            decompress = True
+
```

Comment:
decompress = len(url_parts) > 1 and (url_parts[-1] == "gz")

path.split is typically better handled by pathlib to get an extension
`decompress: bool = Path(path).suffix == "gz"`